rero_ils/modules/circ_policies/api.py

```diff
@@ -292,7 +292,7 @@ def reasons_to_keep(self):
             others['is_default'] = is_default
         return others
 
-    def get_links_to_me(self):
+    def get_links_to_me(self, get_pids=False):
         """Get number of links to policy."""
         links = {}
         return links
```

Comment:
Could you please place a `TODO` comment here to remember to have a better method implementation.